source/includes/builders.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+// start-imports
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Query;
+use MongoDB\Builder\Stage;
+// end-imports
+
+require 'vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+// start-db-coll
+$collection = $client->sample_geospatial->shipwrecks;
+// end-db-coll
+
+// start-find
+/* Creates a query filter by using builders and
+   retrieves matching documents */
+$docs = $collection->find(Query::query(
+    feature_type: Query::eq('Wrecks - Visible'),
+    coordinates: Query::near(
+        Query::geometry(
+            type: 'Point',
+            coordinates: [-79.9, 9.3],
+        ),
+        maxDistance: 10000,
+    )
+));
+
+/* Prints matching documents */
+foreach ($docs as $doc) {
+    echo json_encode($doc), PHP_EOL;
+}
+// end-find
+
+// start-deleteone
+/* Creates a query filter by using builders
+   and deletes the first matching document */
+$result = $collection->deleteOne(Query::query(
+    feature_type: Query::regex('nondangerous$', '')
+));
+
+echo 'Deleted documents: ', $result->getDeletedCount(), PHP_EOL;
+// end-deleteone
+
+// start-updateone
+/* Creates a query filter and an update document by
+   using builders and updates the first matching document */
+$result = $collection->updateOne(
+    Query::query(watlev: Query::eq('partly submerged at high water')),
+    new Pipeline(
+        Stage::set(year: 1870),
+    ),
+);
+
+echo 'Updated documents: ', $result->getModifiedCount(), PHP_EOL;
+// end-updateone
+
+// start-cs
+/* Creates a pipeline to filter for update operations and returns
+   only specific fields */
```

Comment:
Usually, we would use this kind of comment.
```suggestion
// Creates a pipeline to filter for update operations and returns
// only specific fields.
```